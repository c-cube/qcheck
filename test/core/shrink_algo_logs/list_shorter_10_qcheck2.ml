open QCheck2

let list_shorter_10 =
  Test.make ~name:"lists shorter than 10"
    Gen.(list small_int) (Log.shrinks Print.(list int) @@ fun xs -> List.length xs < 10)

let rand = Random.State.make [|1234|];;
try Test.check_exn ~rand list_shorter_10 with Test.Test_fail _ -> ()
