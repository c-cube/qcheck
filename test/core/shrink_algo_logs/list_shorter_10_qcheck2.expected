fails [7; 1; 42; 1; 8; 5; 3; 9; 5; 38; 3; 3; 0; 1; 98; 1; 4; 13; 9; 2; 6; 9; 47; 6; 5; 8; 8; 6; 0; 9; 7; 2; 8; 6; 62; 6; 4; 31; 19; 1; 41; 60; 6; 5; 8; 1; 1; 4; 7; 7; 0; 5; 5; 71; 14; 26; 47; 5; 1; 6; 34; 9; 4; 2; 37; 3; 8; 4; 31; 6; 2; 1; 0; 7; 5; 1; 0; 15; 6; 1; 8; 13; 0; 6; 2; 4; 2; 6; 6; 1; 4; 1; 9; 79; 0; 87; 6; 8; 8; 62; 1; 4; 62; 6; 31; 1; 5; 6; 5; 9; 3; 3; 1; 79; 4; 3; 2; 67; 5; 7; 12; 70; 8; 8; 6; 1; 3; 14; 15; 1; 61; 4; 1; 4; 1; 7; 4; 4; 4; 2; 8; 8; 7; 5; 4; 27; 0; 9; 80; 25; 1; 8; 1; 3; 7; 4; 3; 5; 5; 6; 5; 5; 31; 7; 0; 3; 3; 6; 71; 76; 28; 60; 6; 2; 6; 3; 0; 4; 1; 0; 5; 7; 0; 28; 86; 4; 7; 51; 36; 0; 5; 0; 1; 4; 3; 6; 0; 1; 1; 8; 18; 4; 2; 8; 8; 1; 4; 7; 1; 0; 93; 5; 3; 0; 80; 1; 7; 7; 8; 8; 5; 7; 8; 9; 24; 4; 25; 8; 8; 5; 4; 90; 4; 6; 8; 4; 4; 0; 60; 8; 9; 7; 44; 5; 1; 2; 9; 74; 7; 7]
holds []
fails [9; 61; 8; 4; 69; 2; 0; 72; 4; 41; 3; 8; 75; 83; 7; 7; 3; 9; 5; 8; 4; 1; 0; 2; 1; 4; 7; 6; 2; 1; 4; 86; 3; 79; 7; 86; 52; 39; 19; 0; 4; 7; 7; 7; 0; 4; 8; 8; 0; 5; 13; 1; 5; 0; 7; 12; 64; 34; 1; 1; 85; 8; 2; 9; 76; 0; 2; 5; 76; 69; 8; 8; 0; 1; 2; 2; 4; 60; 29; 5; 9; 4; 0; 8; 3; 3; 5; 1; 35; 8; 2; 7; 23; 61; 56; 8; 1; 1; 78; 7; 5; 0; 30; 9; 3; 7; 28; 57; 98; 3; 52; 3; 82; 7; 5; 5; 6; 8; 1; 6; 8; 9; 8; 16; 0]
holds []
fails [1; 9; 8; 28; 47; 8; 5; 6; 8; 9; 2; 5; 8; 30; 6; 8; 84; 0; 6; 7; 76; 7; 9; 1; 0; 5; 76; 95; 2; 2; 1; 45; 7; 8; 8; 1; 6; 37; 5; 6; 73; 8; 0; 85; 8; 0; 4; 5; 2; 0; 26; 59; 0; 5; 13; 4; 7; 3; 6; 8; 1; 3]
holds []
fails [5; 0; 0; 4; 10; 2; 4; 9; 5; 73; 6; 1; 5; 5; 3; 10; 5; 31; 1; 4; 3; 8; 9; 13; 41; 20; 96; 5; 1; 2; 8]
holds []
fails [9; 8; 73; 5; 8; 2; 1; 8; 2; 6; 4; 18; 5; 76; 3]
holds []
holds [0; 6; 2; 8; 8; 1; 4]
fails [2; 5; 3; 3; 2; 7; 3; 3; 5; 2; 3]
holds []
holds [87; 7; 0; 5; 3]
holds [4; 6; 5; 7; 8; 6; 7; 9]
holds [7; 5; 13; 4; 59; 2; 9; 0; 34]
fails [3; 73; 1; 7; 3; 9; 33; 6; 6; 1]
holds []
holds [13; 6; 4; 0; 4]
holds [9; 7; 98; 0; 7; 9; 4; 4]
holds [47; 38; 7; 2; 2; 2; 3; 0; 5]
fails [0; 73; 1; 7; 3; 9; 33; 6; 6; 1]
fails [0; 0; 1; 7; 3; 9; 33; 6; 6; 1]
fails [0; 0; 0; 7; 3; 9; 33; 6; 6; 1]
fails [0; 0; 0; 0; 3; 9; 33; 6; 6; 1]
fails [0; 0; 0; 0; 0; 9; 33; 6; 6; 1]
fails [0; 0; 0; 0; 0; 0; 33; 6; 6; 1]
fails [0; 0; 0; 0; 0; 0; 0; 6; 6; 1]
fails [0; 0; 0; 0; 0; 0; 0; 0; 6; 1]
fails [0; 0; 0; 0; 0; 0; 0; 0; 0; 1]
fails [0; 0; 0; 0; 0; 0; 0; 0; 0; 0]
